docs/book/releases/v2.3.0.md

```diff
@@ -0,0 +1,84 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.3.0 release
+
+## New Feature
+
+- CSI Migration for in-tree vSphere volumes is promoted to GA. This feature requires upgrading Kubernetes to 1.19 release.
```

Comment:
Can we add  a link to CSI Migration doc here for easier navigation?

Reply:
done